index.rst

```diff
@@ -245,6 +245,97 @@ It is also worth mentioning again that the overall throughput we achieve with Ka
+
+   We do not plan to support LabView for CSCs any longer.
+   The components that are still written in LabView (namely, ATMCS and ATPneumatics) will be converted to TCP/IP and their CSCs ported to salobj.
+
+#. Initial support for CI in jenkins.
+
+   This step is probably going to happen in parallel to the SAL work and will be dedicated to adding support for running CI using the kafka version of our system in our jenkins server.
+
+#. Initial tests on TTS.
+
+   Once SAL C++ is available we can start deploying the TSSW components on TTS and running some initial tracking tests.
+   We can start doing these tests even without the Java libraries (used by the camera) so we hope to focus on the C++ libraries first.
+
+   At this point DDS is still our main development and deployment target, e.g. both ts_salobj and ts_sal will be supporting DDS and Kafka, where DDS will remain the main target.
+   We do not expect developers to migrate to using kafka for development yet, except for a few early adopters.
+
+   However, for this stage of the process we expect to make the initial release candidates for salobj and SAL.
+   
+   Support for CI will still be minimum in jenkins, however since this is supposed to be a drop-in implementation we do not expect any roadblocks.
+   
+   We expect that ts_cycle_build will also be branched out to support building the system for Kafka.
+
+   Because ts_idl will no longer be necessary, it will probably be marked for deprecation at this stage.
+
+#. CI fully supported in jenkins.
+
+   At this point we expect that running CI jobs in jenkins will be fully supported.
+
+   The main focus is to support running jobs for salobj and SAL kafka branches.
+   We also should have in place the release jobs, that will make the SAL libraries available for C++ and Java.
+
+   For Python/salobj, there is no additional support needed as conda packaging will be able to provide all the required artifacts.
+
+#. Initial tests at the summit.
+
+   Once we have certified that we can run the system reliably on TTS, we should schedule testing at the summit.
+   The idea is to deploy the same system that was tested on TTS at the summit and execute a 3-5 nights long AuxTel run as well as any ongoing tests with the SST.
+   If possible it would be desirable to run both telescopes at the same time, driven by the Scheduler.
+   However, we must be attentive to the summit calendar and Kafka testing should be done in a way to minimize impact.
+
+   For this step, the kafka versions of both ts_salobj and ts_sal will be on a separate branch and we should rely on release candidate versions for deployment.
+
+   Development will still target the DDS version of salobj and sal.
+
+#. Adoption evaluation.
+
+   After testing at the summit concludes we will evaluate the results and produce a technote with our findings
+
+   If the test succeeds we can keep the summit running the Kafka version of our stack.
```

Comment:
What does `succeed` mean here? How we will go about making this decision before tech note is written in case of a good outcome?